meson.build

```diff
@@ -36,25 +36,42 @@ if get_option('b_coverage')
 	add_project_arguments(compiler.get_supported_arguments(['-fstack-protector-all', '--param=ssp-buffer-size=4']), language: 'cpp')
 endif
 
-# howardhinnant/date has several names - look for them, from the most
-# to the least explicit name.
-# In Meson 0.60.0, this can be replaced with a simpler:
-#
-#     dependency('howardhinnant-date', 'hinnant-date', 'date')
-#
-date_dep = dependency('howardhinnant-date', required: false)
-if not date_dep.found()
-	date_dep = dependency('hinnant-date', required: false)
-endif
-if not date_dep.found()
-	date_dep = dependency('date', fallback: ['hinnant-date', 'date_dep'])
-endif
-
 deps_libpistache = [
-	dependency('threads'),
-	date_dep
+	dependency('threads')
 ]
 
+#Check if compiler supports C++20 date
+
+cxx_chrono_date_check_code = '''
+#include <sstream>
+#include <chrono>
+int main() {
+    std::stringstream ss;
+	ss << std::format("%FT%T%z", std::chrono::system_clock::now());
```

Comment:
I would avoid using `std::format`, since it isn't well supported everywhere and we're just checking for C++20 chrono anyway. Also, the code isn't ran, so you can simply call the functions you want to check for, without actually writing something that "makes sense".

Reply:
If we want to replace `date::to_stream` from hinnant-date we kinda have no other option. This is the way to turn dates into strings in C++20, no way around it. The `fmt` lib is also available as a sort of preview of C++20 `std::format` like hinnant-date is a preview of the date changes. So a fallback could be introduced, but it would be a rare cornercase for something that is already a rare cornercase (building pistache for C++20).